e2e/testcafe-devextreme/tests/accessibility/accordion.ts

```diff
@@ -0,0 +1,32 @@
+import { Properties } from 'devextreme/ui/accordion.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const items = ['Item_1', 'Item_2', 'Item_3'];
+
+const options: Options<Properties> = {
+  dataSource: [items],
+  collapsible: [true, false],
+  disabled: [true, false],
+  deferRendering: [true, false],
+  hint: [undefined, 'hint'],
+  multiple: [true, false],
```

Comment:
To fully test this functionality, you need to use the created hook to expose multiple items in multiple mode.